node/src/config/arguments.js

```diff
@@ -119,6 +120,13 @@ const argumentsMap = {
     group: ['deploy'],
     description: 'Your git revision, can be a branch tag or a commit hash. Default value "master" '
   },
+  [SKIP_STATE_REFRESH]: {
+    name: SKIP_STATE_REFRESH,
+    alias: 'z',
+    type: String,
```

Comment:
no `	type: bool`?

Reply:
Type bool is checking if the flag exists or not. `env0 -z` is `true` and `env0` is `false`
Using the string we can be more explicit like in `requiresApproval`